sdk/storage/Azure.Storage.Files.DataLake/src/Models/DataLakeServiceProperties.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Azure.Storage.Files.DataLake.Models
+{
+    /// <summary>
+    /// Data Lake Service Properties.
+    /// Note that HNS-enabled storage accounts do not support static website.
+    /// </summary>
+    public class DataLakeServiceProperties
+    {
+        /// <summary>
+        /// Azure Analytics Logging settings.
+        /// </summary>
+        public DataLakeAnalyticsLogging Logging { get; set; }
+
+        /// <summary>
+        /// A summary of request statistics grouped by API in hour or minute aggregates for Data Lake.
+        /// </summary>
+        public DataLakeMetrics HourMetrics { get; set; }
+
+        /// <summary>
+        /// A summary of request statistics grouped by API in hour or minute aggregates for Data Lake.
+        /// </summary>
+        public DataLakeMetrics MinuteMetrics { get; set; }
+
+        /// <summary>
+        /// The set of CORS rules.
+        /// </summary>
+        public IList<DataLakeCorsRule> Cors { get; set; }
+
+        /// <summary>
+        /// The default version to use for requests to the Data Lake service if an incoming request's version is not specified.
+        /// Possible values include version 2008-10-27 and all more recent versions.
+        /// </summary>
+        public string DefaultServiceVersion { get; set; }
+
+        /// <summary>
+        /// The retention policy which determines how long the associated data should persist.
+        /// </summary>
+        public DataLakeRetentionPolicy DeleteRetentionPolicy { get; set; }
```

Comment:
In other places it's easy to figure out what this property mean as they're nested inside some other named part of configuration.
Assuming that this controls how long files/directories exist I think we should be more verbose in naming and doc.
I.e. "The retention policy which determines how long the deleted files and directories should persist."
Maybe this should be called DeletedPathsRetentionPolicy ?

Reply:
This name is pattern-matched from `BlobRetentionPolicy`.